extensions/ql-vscode/src/remote-queries/remote-queries-interface.ts

```diff
@@ -129,6 +135,31 @@ export class RemoteQueriesInterfaceManager {
     });
   }
 
+  private async openFile(filePath: string) {
+    try {
+      const textDocument = await workspace.openTextDocument(filePath);
+      await Window.showTextDocument(textDocument, ViewColumn.One);
```

Comment:
Eventually, it would be nice to do something like https://github.com/github/vscode-codeql/pull/1037, but that can happen later. Once #1037 gets in, we can work on extracting the behaviour so it is used wherever we are opening editors.

Reply:
Oops, good point, thanks for spotting that! I've added some styling, which keeps the text the same colour in all situations.